src/lit-element.ts

```diff
@@ -137,6 +139,18 @@ export class LitElement extends UpdatingElement {
   static hydrate:
       (result: unknown, container: Element|DocumentFragment) => void;
 
+  static get observedAttributes() {
+    return [...super.observedAttributes, 'defer-hydration'];
+  }
+
+  attributeChangedCallback(name: string, old: string | null, value: string | null) {
```

Comment:
I wonder if this should be in UpdatingElement so that it's a good base class for SSR-compatible components, even for other template syntaxes?